spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -113,4 +113,66 @@
     expect(valid_response).to match_response_schema("collection")
     expect(invalid_response).not_to match_response_schema("collection")
   end
+
+  context "when the `required` key is absent" do
+    it "validates with `strict: true`" do
+      create_schema("without_additional", {
+        "type" => "object",
+        "properties" => {
+          "foo" => { "type" => "string" },
+        }
+      })
+      create_schema("with_additional", {
+        "type" => "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.